nms/app/packages/magmalte/app/views/network/FEGNetworkInfo.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+import type {DataRows} from '../../components/DataGrid';
+import type {feg_network} from '@fbcnms/magma-api';
+
+import DataGrid from '../../components/DataGrid';
+import React from 'react';
+
+type Props = {
+  fegNetwork: $Shape<feg_network>,
+};
+
+/**
+ * Returns information about the federation network.
+ * @param {object} props: has a property called fegNetwork that has
+ * information about the federation network.
+ */
+export default function NetworkInfo(props: Props) {
+  const kpiData: DataRows[] = [
+    [
+      {
+        category: 'ID',
+        value: props.fegNetwork.id,
+      },
+    ],
+    [
+      {
+        category: 'Name',
+        value: props.fegNetwork.name,
+      },
+    ],
+    [
+      {
+        category: 'Description',
+        value: props.fegNetwork.description || '-',
+      },
+    ],
+    [
+      {
+        category: 'Federation',
+        value: props.fegNetwork.id,
```

Comment:
why is `props.fegNetwork.id,` two times, here and in line 37?

Reply:
Because the network ID is the same as the federation network id, since it is already a federation network. This would have been different if it was a feg_lte network because the network id is different from the federation network id for that case.

Reply:
If the design mock-up doesn't make sense, we should either change it, or perhaps get some clarification on what was intended.